CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 <!-- ### Fixed -->
 <!-- ### Removed -->
 
+## [Unreleased]
```

Comment:
Ah - I didn't realize that we didn't yet have any changes here. Maybe we can follow the prior convention of commenting this line out & commenting out line 1150.

Then when something is added to Unreleased that person can uncomment the heading & link?
What do you think?

Thanks for the changes!

Reply:
I think it is ok now to have anything there right after a release. The `Unreleased` heading is there so ppl can quickly access all changes (commits) since the last release (tag).

I'd propose to leave it there.

What do you think?